apps/meteor/client/views/admin/users/UsersTable/UsersTable.tsx

```diff
@@ -133,17 +151,34 @@ const UsersTable = ({ reload, tab, onReload }: UsersTableProps): ReactElement |
 					<GenericTableBody>{isLoading && <GenericTableLoadingTable headerCells={5} />}</GenericTableBody>
 				</GenericTable>
 			)}
+
+			{/* TODO: finish error state -> improve tab verification and add translations */}
+			{!isLoading && !isSuccess && tab !== 'all' && tab !== 'active' && (
+				<GenericTable>
+					<GenericTableHeader>{headers}</GenericTableHeader>
+					<GenericTableBody>
+						<States>
+							<StatesIcon name='user' />
+							<StatesTitle>No deactivated users</StatesTitle>
+							<StatesSubtitle>Deactivated users appear here</StatesSubtitle>
+						</States>
+					</GenericTableBody>
+				</GenericTable>
+			)}
+
 			{isSuccess && !!data && !!filteredUsers && data.count > 0 && (
 				<>
 					<GenericTable>
 						<GenericTableHeader>{headers}</GenericTableHeader>
 						<GenericTableBody>
+							{/* TODO: fix user type */}
 							{filteredUsers.map((user) => (
 								<UsersTableRow
 									key={user._id}
 									onClick={handleClickOrKeyDown}
 									mediaQuery={mediaQuery}
-									user={user}
+									user={user as any}
```

Comment:
This problem here was caused due to the change of the type of the users param of the three filter hooks, TS is infering the type of user to be Partial<IUser>[] when in fact it is and shouldnt be this.